langchain/llms/self_hosted_hugging_face.py

```diff
@@ -66,6 +68,8 @@ def _load_transformer(
             model = AutoModelForCausalLM.from_pretrained(model_id, **_model_kwargs)
         elif task == "text2text-generation":
             model = AutoModelForSeq2SeqLM.from_pretrained(model_id, **_model_kwargs)
+        elif task == "summarization":
```

Comment:
We could group these conditions (`elif task in ("text2text-generation", "summarization")`)

Reply:
@vowelparrot I have committed the latest changes based on your valuable advice. Thank you for your guidance and support throughout the review process.